packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -105,6 +113,12 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
         ),
         visitType: visitType,
         location: selectedLocation,
+        attributes: Object.entries(visitAttributes)
```

Comment:
Looks like you'd need to extend the `NewVisitPayload` interface to include `attributes`.

Reply:
I did, but it's still failing in the tsc. It was late yesterday night and I'll just have a look today.
Thanks!